.github/workflow/mvn-pr.yml

```diff
@@ -0,0 +1,35 @@
+name: Signal Server PR Builder
+
+on:
+  pull_request:
+    branches: [ master ]
+    types: [ opened, reopened, synchronize, labeled, unlabeled ]
```

Comment:
I think just using `[ opened, reopened ]` would be enough

Reply:
labeled and unlabeled triggers can be removed.

Reply:
👍🏻